src/locale/languages.ts

```diff
@@ -464,7 +466,8 @@ export const LANGUAGES: Language[] = [
   {code3: 'pli', code2: 'pi', name: 'Pali'},
   {code3: 'pol', code2: 'pl', name: 'Polish'},
   {code3: 'pon', code2: '', name: 'Pohnpeian'},
-  {code3: 'por', code2: 'pt', name: 'Portuguese'},
+  {code3: 'por', code2: 'pt-BR', name: 'Portuguese (BR)'},
```

Comment:
Please do not modify this part, as it refers to the post language, not the app display language. currently, post language does not support regional distinctions.